### ISSUE TITLE
library_if_available() is deprecated

### ISSUE THREAD

**Issue by hadley** [gavinsimpson/coenocliner#23]
```
checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  'citation()' on how to cite R or R packages in publications.
  
  Type 'demo()' for some demos, 'help()' for on-line help, or
  'help.start()' for an HTML browser interface to help.
  Type 'q()' to quit R.
  
  > ## Test `coenocliner` using the `testthat` package
  > 
  > ## Setup
  > library("testthat")
  > library_if_available("coenocliner")
  Error: could not find function "library_if_available"
  Execution halted
```

**Comment by gavinsimpson** [comment]
Thanks @hadley I'll fix this and be ready to push a release to CRAN.

**Comment by gavinsimpson** [comment]
Fixed via 6923f1e503cfa5f64bb41d582d55259471fe727a
